go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/Cloud-Foundations/keymaster
 
-go 1.18
+go 1.21
```

Comment:
Do we need to force the most recent version of Go? Is 1.19 sufficient?

Reply:
unfortunatelly, the library I chose requires 1.21 (1.22 is the lastest)